week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -1,34 +1,29 @@
 /*
 Many years into the future, a team of Space Voyagers find their ship is low on Oxygen and need to dock
 somewhere safe while they call home for help.
-  
 Their computer detects a list of nearby planets that have Oxygen in their atmosphere.
-
 To be safe, they need to land on the first unamed planet that has Oxygen levels between 19.5% and 23.5%.
-
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
-
 Some string methods that might help you here are .replace() and .substring(). Let's look at a quick
 example before trying the exercise.
 */
-
 /* .replace() allows us to add something where we removed something*/
 let greeting = "Good Morning";
 greeting.replace('Morning', 'Evening'); // outputs Good Evening
-
-
 /* .substring() allows us to remove things from strings */
 let dessert = "ice cream and pancakes";
-
 let newdessert = dessert.substring(0, 9);
-
 console.log(newdessert); // returns ice cream
 
 /* + + + + + + + */
 /* Now try the exercise */
 
 function safeLevels() {
-
+  if (oxygenLevels1 >= "19.5%" && oxygenLevels1 <= "23.5%") {
```

Comment:
`oxygenLevels1` should have been an input parameter instead of using the global variable from outside.

Reply:
`oxygenLevels1 >= "19.5%"` comparison won't work because `oxygenLevels1` is an array whereas "19.5%" is a string. Also, you have to parse number form the value of each array element and then compare as a number.